SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Supported Versions
+
+SpeechBrain aims to support the latest major version (at x.x level) with security updates.
```

Comment:
"(at x.x level)" ?

Reply:
Ah maybe x.y is better. I mean like the latest 0.5 will be supported until we release 0.6, and then that will be the one we focus on.